src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/AbstractReferenceFinder.cs

```diff
@@ -128,7 +142,7 @@ protected Task<ImmutableArray<Document>> FindDocumentsAsync(Project project, IIm
             return FindDocumentsAsync(project, documents, async (d, c) =>
             {
                 var info = await SyntaxTreeIndex.GetIndexAsync(d, c).ConfigureAwait(false);
-                return info.ContainsPredefinedOperator(op);
+                return info.ContainsPredefinedOperator(op) || info.ContainsGlobalAttributes;
```

Comment:
,ight be worth a comment that an operator can be references in a global suppress attribute.